foreman.te

```diff
@@ -283,6 +290,10 @@ optional_policy(`
     mta_send_mail(passenger_t)
 ')
 
+tunable_policy(`passenger_can_connect_smtp',`
+    corenet_tcp_connect_smtp_port(passenger_t)
```

Comment:
It appears so.

<pre>
# semanage port -l | grep smtp_port_t
smtp_port_t                    tcp      25, 465, 587
</pre>